movies/views.py

```
@@ -26,7 +26,22 @@ class RetrieveUpdateDestroyMovieAPIView(RetrieveUpdateDestroyAPIView):
     queryset = Movie.objects.all()
     permission_classes = [IsAuthenticated, IsOwnerOrReadOnly]
 
-
-
-
-
+class TheatreView(APIView):
```

Ruff (0.8.2)

Undefined name `APIView`

(F821)